armory/baseline_models/pytorch/cifar.py

```diff
@@ -46,10 +46,6 @@ def make_cifar_model(**kwargs):
 
 def get_art_model(model_kwargs, wrapper_kwargs, weights_file=None):
     model = make_cifar_model(**model_kwargs)
-
-    if torch.cuda.is_available():
-        model.cuda()
```

Comment:
Reason to remove these? We do need to change this method to match the way it's device placement is done, but probably not this PR